core/src/main/scala/org/apache/spark/deploy/master/WorkerInfo.scala

```diff
@@ -107,4 +107,8 @@ private[spark] class WorkerInfo(
   def setState(state: WorkerState.Value): Unit = {
     this.state = state
   }
+
+  def isAlive(): Boolean={
```

Comment:
The reason why I write a method here is due to JsonProtocol.scala  cannot access   WorkerState.ALIVE. 

Changing the WorkerState from private to other permission is not good approach. I think it is the best way to keep the current class/namespace struct and get thing done.